package.json

```diff
@@ -1,22 +1,31 @@
 {
-  "name": "@squadcraft/eslint-config",
+  "name": "@squadcraft/eslint-config-squadcraft",
   "version": "1.0.0",
   "description": "eslint for squadcraft",
-  "main": "lib/index.js",
+  "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/arturwita/eslint-config.git"
+    "url": "git+https://github.com/arturwita/eslint-config-squadcraft.git"
   },
   "keywords": [
-    "eslint"
+    "eslint",
+    "eslint-config",
+    "eslint-config-squadcraft",
+    "squadcraft"
   ],
   "author": "arturwita",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/arturwita/eslint-config/issues"
+    "url": "https://github.com/arturwita/eslint-config-squadcraft/issues"
   },
-  "homepage": "https://github.com/arturwita/eslint-config#readme"
+  "homepage": "https://github.com/arturwita/eslint-config-squadcraft#readme",
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-node": "^11.0.0"
+  }
```

Comment:
this time I'm pretty sure those are dependencies, but please check if it can be dev-deps :)

Reply:
check out airbnb's config-base [here](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json). They have both eslint and eslint plugins stored in dev-deps.

Reply:
yes, but then - will the package work in main repo, where main repo doesn't have eslint installed separately? check it out